### PR TITLE
Pr/clean up logs

### PR DESCRIPTION
**Pull request by fildon**
**What**:
Adds a bot log while cleaning up meetup channels

**Why**:
This is a first step in investigating the issue raised here:
https://github.com/kentcdodds/kcd-discord-bot/issues/72

**How**:
Using the existing botlog utility

**Checklist**:

- [ ] Documentation N/A
- [ ] Tests
- [X] Ready to be merged

I've not added tests, but I'm not convinced it's necessary for a log statement.

### CODE REVIEW

File: src/meetup/cleanup.ts [hunk 1]
```diff
@@ -113,10 +114,14 @@ async function cleanup(guild: TDiscord.Guild) {
...
-      meetupChannel.createdAt.getTime() < cutoffAge &&
-      meetupChannel.members.size === 0
-    ) {
+    const createdAt = meetupChannel.createdAt.getTime()
+    const memberCount = meetupChannel.members.size
+    if (createdAt < cutoffAge && memberCount === 0) {
+      botLog(
+        guild,
+        () =>
+          `Cleaning up channel ${meetupChannel.name} created at: ${createdAt}`,
```
Comment (Aprillion) [inline src/meetup/cleanup.ts]: I would explicitly log what the code believe is "now", not just what the logger appends 😓 

```suggestion
          `Cleaning up channel ${meetupChannel.name} created at: ${createdAt}, now: ${now}`,
```

**Review by Aprillion — APPROVED** [review]

**Review by kentcdodds — APPROVED** [review]
Great, thanks!

**Comment by kentcdodds** [comment]
@all-contributors please add @Aprillion for review

**Comment by kentcdodds** [comment]
@all-contributors please add @fildon for code

**Comment by kentcdodds** [comment]
🤦‍♂️

**Pull request merged**